src/styles/card.scss

```diff
@@ -0,0 +1,84 @@
+
+.card {
+  box-sizing: border-box;
+  width: $card-width;
+  padding: 32px 16px 16px;
+  font-family: Roboto, sans-serif;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+
+  }
+
+  .name {
+    color: $main-text-color;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin-top: 40px;
+    margin-bottom: 0;
+  }
+
+  .code {
+    color: $secondary-text-color;
+    font-size: 10px;
+    line-height: 14px;
+    margin-top: 4px;
+  }
+
+  &__rating {
+    margin-top: 16px;
+
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__reviews {
+    color: $main-text-color;
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &__cost {
+    margin-top: 24px;
+    display: flex;
+    justify-content: space-between;
+  }
+
+  &__price {
+    color: $secondary-text-color;
+    font-size: 12px;
+    line-height: 18px;
+  }
+
+  &__money {
```

Comment:
The class name should follow BEM naming convention. Please change '&__money' to something more descriptive like '&__price-amount'.